### PR TITLE
Homepage structure

### LINKED ISSUE #15: [4pt]  - Display list of items on the Homepage - Student A
Create the main part of the homepage that keeps the layout from the wireframe: 

Screenshot 2021-06-27 at 21 50 23

When the page loads, the webapp retrieves data from  the selected API and shows the list of items on screen with the basic data (e.g. title + image).

Prepare also "Comments" and "Reservations" buttons. They should be doing nothing - just being displayed.

This task **does not** include displaying the number of likes for each item.

This task **does not** include the counter of the items.

### PR DESCRIPTION
**Pull request by sntgopl**
In this milestone I have:
- Created a new branch for working on the homepage.
- Created HTML structure for the home section displaying 6 cards with flex and wrap.
- Gave CSS structure to correct the position of each card and inside of each.
- Uploaded demo image to try positioning the card image.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,19 +1,65 @@
...
+			<div class="card">
+					<img class="pokemon-img" src="./images/charmander.jpeg" alt="charmander">
+					<div class="card-data">
+						<h2 class="pokemon-name">Pokemon 1</h2>
+						<div class="like-section"><i class="fa-regular fa-heart heart"></i><p>5 likes</p></div>
+				</div>
+					<button class="comment-button">Comments</button>
+			</div>
+	</section>
+	<section id="popUp">
```
Comment (Camilovelag) [inline src/index.html:68]: [OPTIONAL]  Please kindly change the `section` id to lowercase to match the naming conventions for CSS.

**Review by Camilovelag — APPROVED** [review]
## Hi Santiago 👋 

I think the milestone was _successfully completed_. I added only optional requirements (check comments named _[OPTIONAL]_.

good_job

Happy coding 👏👏👏

File: src/index.html (hunk 1 above)
Comment (sntgopl) [reply src/index.html:61]: Thank you Camilo for your comment. I will commit the changes and merge them to the dev branch.

**Comment by sntgopl** [comment]
Thank you Camilo for your comment. I will commit the changes and merge them to the dev branch.

**Pull request merged**